### PR TITLE
Adding validator for required fields

### PR DESCRIPTION
**Pull request by MelsHyrule**
Related to https://github.com/ManageIQ/manageiq-ui-classic/pull/8339

@miq-bot add-reviewer @nasark 
@miq-bot add-reviewer @kbrock 
@miq-bot assign @kbrock

### CODE REVIEW

**Review by nasark — APPROVED** [review]

File: app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb [hunk 1]
```diff
@@ -44,6 +46,7 @@ def self.params_for_create
           :label        => _('Cloud Database'),
           :includeEmpty => true,
           :isRequired   => true,
+          :validate   => [{:type => 'required'}],
           :options      => ["aurora", "aurora-mysql", "aurora-postgresql", "mariadb", "postgres", "mysql", "oracle-ee", "oracle-ee-cdb", "oracle-se2", "oracle-se2-cdb", "sqlserver-ee", "sqlserver-se", "sqlserver-ex", "sqlserver-web"].map do |db|
```
Comment (Fryguy) [inline app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb:50]: @GilbertCherrie Does this one suffer from the same drop-down problem we noticed elsewhere?

Comment (GilbertCherrie) [reply app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb:50]: It shouldn't, that drop down issue only affects selects that have `isSearchable: true` or `isClearable: true`.

**Review by agrare — APPROVED** [review]

File: app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb [hunk 2]
```diff
@@ -30,6 +31,7 @@ def self.params_for_create
           :label        => _('Cloud Database Instance Class'),
           :includeEmpty => true,
           :isRequired   => true,
+          :validate   => [{:type => 'required'}],
```
Comment (agrare) [inline app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb:34]: Actually just noticed the rubocop warnings still aren't addressed, can you fix those?
```suggestion
          :validate     => [{:type => 'required'}],
```

File: app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb [hunk 3]
```diff
@@ -44,6 +46,7 @@ def self.params_for_create
           :label        => _('Cloud Database'),
           :includeEmpty => true,
           :isRequired   => true,
+          :validate   => [{:type => 'required'}],
```
Comment (agrare) [inline app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb:49]:
```suggestion
          :validate     => [{:type => 'required'}],
```

**Comment by MelsHyrule** [comment]
@agrare applied the changes you asked, i think it's ready to merge now

**Pull request merged**